app/lib/features/chat_ng/widgets/events/chat_event.dart

```diff
@@ -0,0 +1,199 @@
+import 'package:acter/common/providers/common_providers.dart';
+import 'package:acter/common/providers/room_providers.dart';
+import 'package:acter/features/chat/widgets/messages/encrypted_message.dart';
+import 'package:acter/features/chat/widgets/messages/redacted_message.dart';
+import 'package:acter/features/chat_ng/providers/chat_room_messages_provider.dart';
+import 'package:acter/features/chat_ng/widgets/chat_bubble.dart';
+import 'package:acter/features/chat_ng/widgets/events/file_message_event.dart';
+import 'package:acter/features/chat_ng/widgets/events/image_message_event.dart';
+import 'package:acter/features/chat_ng/widgets/events/member_update_event.dart';
+import 'package:acter/features/chat_ng/widgets/events/state_update_event.dart';
+import 'package:acter/features/chat_ng/widgets/events/text_message_event.dart';
+import 'package:acter/features/chat_ng/widgets/events/video_message_event.dart';
+import 'package:acter_avatar/acter_avatar.dart';
+import 'package:flutter/material.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart'
+    show RoomMessage, RoomVirtualItem, RoomEventItem;
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:logging/logging.dart';
+
+final _log = Logger('a3::chat::widgets::room_message');
+
+class ChatEvent extends ConsumerWidget {
+  final String roomId;
+  final String eventId;
+
+  const ChatEvent({
+    super.key,
+    required this.roomId,
+    required this.eventId,
+  });
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final msg = ref.watch(chatRoomMessageProvider((roomId, eventId)));
+
+    if (msg == null) {
+      _log.severe('Msg not found $roomId $eventId');
+      return const SizedBox.shrink();
```

Comment:
this is such a severe inconsistency problem in our code, I'd argue that we should be showing an error message to the user as well.